docker-compose.yml

```diff
@@ -118,6 +118,7 @@ services:
   '7.4-debug': { <<: *base_php_service, image: 'datadog/dd-trace-ci:php-7.4-debug-alpine-3.11' }
   '7.4-debug-asan': { <<: *base_php_service, image: 'datadog/dd-trace-ci:php-7.4-debug-asan-buster' }
   '8.0': { <<: *base_php_service, image: 'datadog/dd-trace-ci:php-8.0_buster' }
+  '8.0-shared-ext': { <<: *base_php_service, image: 'datadog/dd-trace-ci:php-8.0-shared-ext_buster' }
```

Comment:
Here we 
```suggestion
  '8.0-shared-ext': { <<: *base_php_service, image: 'datadog/dd-trace-ci:php-8.0-shared-ext' }
```

Reply:
Ah. I meant to push the container up as `datadog/dd-trace-ci:php-8.0-shared-ext_buster` originally but I accidentally truncated it when playing with a variation. We'll just go with the truncated version since that's the one that is pushed up. Thanks for noticing this!